core-bundle/webpack.config.js

```diff
@@ -0,0 +1,16 @@
+const Encore = require('@symfony/webpack-encore');
+
+Encore
+    .setOutputPath('public/')
+    .setPublicPath('/bundles/contaocore')
+    .setManifestKeyPrefix('')
+    .cleanupOutputBeforeBuild(['**/*', '!core.**', '!mootao.**'])
+    .disableSingleRuntimeChunk()
+
+    .enableSourceMaps()
+    .enableVersioning()
+
```

Comment:
I'm unsure about the babel defaults. Should we create a `babel.config.js` or do we need to enable `babel/preset-env` polyfills here?

```js
    .configureBabelPresetEnv((config) => {
        config.useBuiltIns = 'usage';
        config.corejs = 3;
        // …
    })
```

Reply:
according to https://symfony.com/doc/current/frontend/encore/babel.html that is not necessary

Reply:
Ah nice, I didn't know that!